kratos/geometries/geometry.h

```diff
@@ -731,6 +731,54 @@ class Geometry
     ///@name Inquiry
     ///@{
 
+    /**
+     * @brief Checks if two GeometryType have the same type
+     * @return True if the objects are the same type, false otherwise
+     */
+    inline static bool HasSameType(const GeometryType& rLHS, const GeometryType& rRHS) {
+        return (typeid(rLHS) == typeid(rRHS));
```

Comment:
this requires the `type_info` include

Reply:
from [cppreference](https://en.cppreference.com/w/cpp/language/typeid):
The header <typeinfo> must be included before using typeid (if the header is not included, every use of the keyword typeid makes the program ill-formed.)

Reply:
Done